src/app/api/webhooks/stripe/route.ts

```
@@ -0,0 +1,185 @@
+      { status: 500 }
+    )
+  }
+}
+
+async function handleStripeEvent(event: Stripe.Event) {
+  const session = event.data.object as Stripe.Checkout.Session | Stripe.Subscription | Stripe.Invoice
+
+  switch (event.type) {
+    case 'checkout.session.completed':
+      const completedSession = session as Stripe.Checkout.Session
+      await handleCheckoutCompleted(completedSession)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)